.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,3 +18,7 @@ This PR closes #{issue_number}.
 
 * [ ] I added tests for new functionality or bugfixes.
 * [ ] I ran `task all`, and it passed.
+* [ ] I added/updated comments for both exported and unexported top-level declarations.
```

Comment:
I suggest adding an explanation here too, as `top-level declarations` is a bit confusing from the first read.
```suggestion
* [ ] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
```